scala-setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+set -eu
+
+SCALA_CLI_BASE_URL="https://github.com/VirtuslabRnD/scala-cli/releases/download/latest/"
```

Comment:
I just deleted the `latest` tag, so that GitHub "latest" redirects work. The following now works (and downloads the actual latest release):
```suggestion
SCALA_CLI_BASE_URL="https://github.com/VirtuslabRnD/scala-cli/releases/latest/download/"
```